docs/docs/adrs/adr-015-partial-set-security.md

```diff
@@ -0,0 +1,224 @@
+Third and for the above reasons, it is challenging for a new consumer chain to join Replicated Security.
+
+As a solution, we present _Partial Set Security_ (PSS). As the name suggests, PSS allows for every consumer chain to be secured by only a subset of the provider validator set. 
+In what follows we propose the exact steps we need to take to implement PSS. This is a first iteration of PSS, and therefore we present the most minimal solution that make PSS possible.
+
+## Decision
+In Replicated Security, all the provider validators have to secure every consumer chain (with the exception of those validators allowed to opt out through the [soft opt-out](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/adr-009-soft-opt-out.md) feature). 
+In PSS, we introduce a parameter `N` for each consumer chain and require that the validators in top `N%` of the provider's voting power have to secure the consumer chain. 
+Additionally, we allow the validators outside of the top `N%` to dynamically opt in if they want to validate on the consumer chain.
+For example, if a consumer chain has `N = 95%`, then it ultimately receives the same security it receives today with Replicated Security (with a default [SoftOptOutThreshold](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/adr-009-soft-opt-out.md) of 5%).
+On the other hand, if a consumer chain has `N = 0%`, then no validator is forced to validate the chain, but validators can opt in to do so instead.
+
+For the remainder of this ADR, we call a consumer chain _Top N_ if it has joined as a Top N chain with `N > 0` and _Opt In_ chain otherwise.  An Opt In consumer chain is secured only by the validators that have opted in to secure that chain. 
+
+We intend to implement PSS using a feature branch off [v4.0.0 interchain security](https://github.com/cosmos/interchain-security/tree/v4.0.0).
+
+### How do consumer chains join?
+As a simplification and to avoid [chain id squatting](https://forum.cosmos.network/t/pss-permissionless-vs-premissioned-lite-opt-in-consumer-chains/12984/17), a consumer chain can only join PSS through a governance proposal and not in a permissionless way.
+
+Consumer chains join PSS the same way chains now join Replicated Security, namely through a `ConsumerAdditionProposal` proposal.
+We extend [`ConsumerAdditionProposal`](https://github.com/cosmos/interchain-security/blob/v4.0.0/proto/interchain_security/ccv/provider/v1/provider.proto#L27) with one optional field:
+
+`uint32 top_N`: Corresponds to the percentage of validators that join under the Top N case.
+For example, `53` corresponds to a Top 53% chain, meaning that the top `53%` provider validators have to validate the proposed consumer chain.
+`top_N`  can be `0` or include any value in `[50, 95]`. A chain can join with `top_N == 0` as an Opt In, or with `top_N ∈ [50, 95]` as a Top N chain.
+
+In case of a Top N chain, we restrict the possible values of `top_N` from `(0, 100]` to `[50, 95]`.
```

Comment:
It seems then that we do not really need `SoftOptOutThreshold` and it would make sense to just have `top_N` be anything between `[50, 100]`. If a consumer chain is added with `top_N = 95` then it's like `SoftOptOutThreshold` was set to 5, if a consumer chain is set with `top_N = 90`, then it's like `SoftOptOutThreshold` was set to 10.

Reply:
Updated the ADR to capture the above. Resolving this.